ERCS/erc-7820.md

```diff
@@ -0,0 +1,411 @@
+---
+eip: 7820
+title: Access Control Registry
+description: The AccessControlRegistry contract standardizes access control management by allowing registration, unregistration, role assignment, and role revocation for contracts, ensuring secure and transparent role management.
```

Comment:
I think this goes over the character limit for this field (the bot will let us know in a second.) If it does, try to remove extra fluff. Maybe:

```suggestion
description: Registration, unregistration, role assignment, and role revocation for contracts, ensuring secure and transparent role management.
```